dev-docs/show-multiformat-ads.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: page
+title: Show Multiformat Ads with Prebid.js
+description: Show Multiformat Ads with Prebid.js
+pid: 0
+is_top_nav: yeah
+top_nav_section: dev_docs
+nav_section: prebid-multiformat
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Show Multiformat Ads with Prebid.js
+{: .no_toc }
+
+This page has instructions for showing multiformat ads using Prebid.js.
+
+Multiformat ads allow you to declare multiple media types on a single ad unit.  For example, you can set up one ad unit that could show a banner, native, or outstream video ad, depending on which had the highest bid. 
+
+Once declared, any bidder that supports at least one of the media types can participate in the auction for that ad unit.
+
+{: .alert.alert-info :}
+For ad ops setup instructions, see [Setting up Prebid Multiformat in DFP]({{site.baseurl}}/adops/setting-up-prebid-multiformat-in-dfp.html).
+
+* TOC
+{:toc}
+
+## How Multiformat Ads Work
+
+Multiformat ads allow you to say that a single ad unit may be filled by any eligible banner, video, or native ad.
```

Comment:
"Prebid multi-format ad units allow direct competition between banner, native, and/ or video bids".

Reply:
Thanks, updated in be9fd69